YoutubeExplodeTests/YoutubeClientPlaylistParsingTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using YoutubeExplode.Services;
+
+namespace YoutubeExplode.Tests
+{
+    [TestClass]
+    public class YoutubeClientPlaylistParsingTests
```

Comment:
I see your point. Ok, I will remove this and add a new test to the integration tests.